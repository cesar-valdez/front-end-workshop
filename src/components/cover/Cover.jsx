import React from 'react';

import './cover.scss';

import logo from './images/logo.svg';
import puppinessOGV from './video/puppiness.ogv';
import puppinessWEBM from './video/puppiness.webm';
import puppinessMP4 from './video/puppiness.mp4';

const Cover = () => (
  <div className="yta_Cover">
    <img src={logo} className="yta_Cover-logo" alt="Hogares para nuestros amigos" />
    <video autoPlay muted loop className="yta_Cover-video">
      <source src={puppinessOGV} type="video/ogg" />
      <source src={puppinessWEBM} type="video/webm" />
      <source src={puppinessMP4} type="video/mp4" />
    </video>
  </div>
)

export default Cover;