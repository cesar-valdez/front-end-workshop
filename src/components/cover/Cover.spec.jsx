import React from 'react';
import { shallow } from 'enzyme';

import Cover from './Cover';

jest.mock('./images/logo.svg', () => './images/logo.svg');
jest.mock('./video/puppiness.ogv', () => './video/puppiness.ogv');
jest.mock('./video/puppiness.webm', () => './video/puppiness.webm');
jest.mock('./video/puppiness.mp4', () => './video/puppiness.mp4');

describe('Cover', () => {
    const cover = shallow(<Cover />);
    it('should render Cover component', () => {
        expect(cover).toMatchSnapshot();
    });
});