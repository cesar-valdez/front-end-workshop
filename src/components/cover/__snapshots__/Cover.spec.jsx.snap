// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover should render Cover component 1`] = `
<div
  className="yta_Cover"
>
  <img
    alt="Hogares para nuestros amigos"
    className="yta_Cover-logo"
    src="./images/logo.svg"
  />
  <video
    autoPlay={true}
    className="yta_Cover-video"
    loop={true}
    muted={true}
  >
    <source
      src="./video/puppiness.ogv"
      type="video/ogg"
    />
    <source
      src="./video/puppiness.webm"
      type="video/webm"
    />
    <source
      src="./video/puppiness.mp4"
      type="video/mp4"
    />
  </video>
</div>
`;
